recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+MACOSX_DEPLOYMENT_TARGET:  # [osx and x86]
+  - '10.12'                # [osx and x86]
```

Comment:
This needs a `run` dependency on `__osx` as outlined in https://conda-forge.org/news/2024/01/29/rust-175-osx-sdk/

Reply:
Implemented on 6543e96